packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -236,14 +275,27 @@ export const getExplorerUrl = (chainId: ChainId) => {
 export const getL1BlockTime = (chainId: number) => {
   const chain = getChainByChainId(getBaseChainIdByChainId({ chainId }))
 
-  if (!chain || !isL1Chain(chain)) {
+  if (!chain || (!isL1Chain(chain) && !isBaseChain(chain))) {
     throw new Error(`Couldn't get block time. Unexpected chain ID: ${chainId}`)
   }
 
+  const { isBase } = isNetwork(chainId)
+
+  if (isBase) {
+    // For Arbitrum L3s built on top of an OP Stack L2, `block.number` will return the L2 block number.
+    // L2 blocks in OP Stack chains are produced every 2 seconds
+    return 2
```

Comment:
We store this in the `BaseNetwork` object, can we return this instead?
```typescript
return chain.blockTime
```

And then we don't need the `isBase` chain at all?